programming-mission/how-tos.md

```diff
@@ -1,50 +0,0 @@
-## How to create a new Instant Answer
-1. [Create an IA Page](https://duck.co/ia/new_ia)
-2. Follow the steps in "How to open a Pull Request"
-
-## How to report a bug
-1. Follow the steps in "How to create a GitHub issue"
-2. Add "Bug: " before the title
-
-## How to create a GitHub issue
-1. Visit the IA Page and clicking the "Create Issue" button. 
-2. Add the issue description following the template
-
-#### Description
-Divide the description into sections:
-
-- "Background", similar to the forum topic, explaining what the issue is and why it needs to be addressed that way
-- "To Do", a simple list of steps to follow in order to complete the task
-- "Forum", a paragraph containing a link to the Programming mission Overview topic and to the forum topic to which that issue belongs.
-
-#### Examples
-[MDN JavaScript: Expressions and operators section](https://github.com/duckduckgo/zeroclickinfo-fathead/issues/330)
-
-## How to open a Pull Request
-1. Go to the GitHub repository for the Instant Answer type you're building and open a Pull Request to master against your working branch
-2. Fill in the Pull Request template
-3. Add the link to the IA Page
-4. Add the link to the forum topic for the specific project
-
-## How to open a Fathead Pull Request 
-1. In your working branch, create a file listing all the basic methods, keywords and operators for your language and commit it (find more details in the docs: [Creating Effective Fatheads](https://docs.duckduckhack.com/resources/creating-effective-fatheads.html))
-2. Follow the steps listed in "How to open a Pull Request"
-
-## How to tell if a Pull Request can be merged
```

Comment:
Seems like most of this is still useful outside of the PM? Can we pull out only PM stuff and re-name + move it?